src/test/java/Homework19Test.java

```diff
@@ -0,0 +1,40 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework19Test extends BaseTest {
+
+
+    @Test(description = "Delete an Empty PlayList")
+    public void deleteEmptyPlaylist(){
+        logIn("skyeman75@gmail.com", "te$t$tudent");
+        verifyLogIn();
+        openPlayList();
+        deletePlayList();
+        WebElement deletedPlaylistMessage = getDeletedPlaylistMessage();
+        Assert.assertTrue(deletedPlaylistMessage.isDisplayed());
+    }
+
+    public void openPlayList(){
+        wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector(".playlist:nth-child(3)"))).click();
+        }
+    private void deletePlayList(){
+        wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector(".btn-delete-playlist"))).click();
+
+
+    }
+    public WebElement getDeletedPlaylistMessage(){
+
+        return driver.findElement(By.cssSelector("div.success.show"));
+    }
+
+
+}
+
+
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._